src/lib/LUA/rx_devLUA.cpp

```diff
@@ -627,6 +667,21 @@ static int event()
   }
   setLuaTextSelectionValue(&luaBindStorage, config.GetBindStorage());
   updateBindModeLabel();
+
+  if (config.GetSourceSysId() == 0) 
+  {
+    setLuaUint8Value(&luaSourceSysId, 255);
+  } else {
```

Comment:
We generally have newlines around the else's. We're loosely following the MS code format.
```cpp
}
else
{
```

Reply:
would using ternary be a good alternative?
I don't think adding way more lines to simple override will significantly improve readability here.
```CPP
setLuaUint8Value(&luaSourceSysId, config.GetSourceSysId() == 0 ? 255 : config.GetSourceSysId());
```

Reply:
Totally fine with that.

Reply:
done.